tests/codegen/loads.rs

```diff
@@ -1,4 +1,6 @@
 //@ compile-flags: -C no-prepopulate-passes -Zmir-opt-level=0
+//@ ignore-debug: We don't emit load metadata in debug
```

Comment:
ignore-debug tests are annoying as they don't run in PR CI.

Can't this just set the `-O` flag to force an optimized build?